### PR TITLE
Response to reviewer suggestions for reorg of subunit vaccines

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

- [x] 1.6 Sections 6.2 and 6.3 (starting from Line 465): While in section 5.1, authors provided more thorough reviews and went over many different IWV-based COVID-19 vaccines, only two specific subunit vaccines, Nuvaxoid and Covifenz, were mentioned. Please consider including more subunit vaccines in Table 2 in the paragraphs and make them concise, as needed. This change may also impact Section 6.4, so please update Section 6.4 accordingly.
- [x] 3.1 The manuscript is subdivided into sections and sub-sections with headings and subheadings, respectively. Please ensure that the text under each subsection is reflecting the heading/subheading and that the information the reader intuitively seeks for under a specific heading is included in that heading. 
- [x] If such information/data are not or not yet available, I would recommend mentioning the lack of such. For example(s), certain sections such as 6.4 the authors revisit clinical trials of Novavax and do not discuss safety.
- [x] Please make the distinctions between trials and real-world data clear throughout the manuscript.
- [x] 3.2 As I understood, the authors generally aim to discuss the platform, dose/regimen, clinical trial efficacy and safety data in sections named Application to COVID-19 or ones with vaccine name (Covifenz, Nuvaxovid) and then RW effectiveness, safety and response/implications for VoC under sections named Real-World Safety and Efficacy. I would first suggest using a uniform naming system for different platforms. Some Real-World Safety and Efficacy sections go back to clinical trials and overview VoC, some are entirely dedicated to VoC. You could add VoC to the subheadings or, as with live attenuated vaccines section, have a separate subsection devoted to VoC, as large part of VoC-related text represents immune outcome data, not RWE per se. With, for example, CoronaVac, the real-world data from Chile is addressed in the more general section and not mentioned in the RW Safety and Efficacy section. As above, I would recommend clearly separating the source of data (trial vs real-world). Also, I suggest logically structuring the RW Safety and Efficacy section as follows: first - effectiveness data (or lack of such) shortly following vaccination, over time, as related to boosters, as pertaining to the VoC.
- [x] 3.3 The authors discuss efficacy/effectiveness, side effect profile, sometimes regulatory details for most of the vaccines. Please make sure each of these outcomes are addressed for each vaccine. For example, RW safety and effectiveness information is lacking for IWV vaccines section, while all of them discuss VoC. Similarly, **safety and side effect details are not provided for Nuvaxovid, for example**. As above, I recommend ensuring the data points given for each platform/vaccine are uniform; separating immune outcome data from the real-world effectiveness; mentioning the lack of data wherever relevant.
- [x] 3.6 When providing percent-efficacy or effectiveness, I recommend to consistently indicate the time period/interval over which the outcome is reported (already included in many cases).
- [x] 3.7 I would suggest using the same naming for vaccines after they (and their alternative names) are first mentioned. For example, Covilo and Sinopharm, CoronaVac and Sinovac are used interchangeably, and it can be confusing for some readers.
- [x] 3.9 Please make sure all relevant sentences are followed by a reference, where appropriate, e.g. (not limited to) lines 143, 191, **469**.
- [x] 3.30 Lines 409-411 are identical to lines 415-417, please leave where more appropriate.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/22.vaccines-trad.md [hunk 1]
```diff
@@ -424,31 +425,104 @@ These data are retrieved from Our World in Data [@url:https://ourworldindata.org
...
+However, following the delays, Gavi (which oversees COVAX) terminated the agreement, leading to ongoing legal disagreements between the nonprofit and Novavax las of late 2022 [@url:https://www.reuters.com/business/healthcare-pharmaceuticals/gavi-says-it-is-not-breach-novavax-vaccine-deal-2022-11-22; @url:https://www.cnbc.com/2022/05/13/novavax-confident-covid-vaccine-will-receive-fda-authorization-in-june-after-delays.html].
+
+As with other vaccines, the question of how well Nuvaxovid continues to provide protection as SARS-CoV-2 evolves has been raised.
+_Post hoc_ analysis in the phase III trial indicated a VE of 86.3% against the Alpha variant (identified based on the presence of the 69–70del polymorphism) and 96.4% against viral specimens lacking the 69-70del polymorphism [@doi:10.1056/NEJMoa2107659].
+In the second phase III trial [@doi:10.1101/2021.10.05.21264567], whole-genome sequencing was obtained from 61 of the 77 observed cases, and 79% of infections were identified as a VOC or variant of interest (VOI) known at the time of the study.
```
Comment (RLordan) [inline content/22.vaccines-trad.md:467]: Is VOI necessary here?

File: content/22.vaccines-trad.md [hunk 2]
```diff
@@ -424,31 +425,104 @@ These data are retrieved from Our World in Data [@url:https://ourworldindata.org
...
+The combination appeared to be safe and effective in preclinical studies [@doi:10.1101/2021.05.05.442782].
+
+#### The Cuban Center for Genetic Engineering and Biotechnology's Abdala Vaccine
+
+Another notable protein subunit vaccine development program came out of Cuba.
+Concerned about their ability to access vaccines, especially given the U.S.'s embargo [@doi:10.1038/d41586-021-03470-x], health officials in this developing country made the decision in March 2020 to undertake vaccine development on their own [@doi:10.37757/mr2022.v24.n3-4.13].
+Today, three Cuban protein subunit vaccines have been approved for use: Abdala, which was developed at the Cuban Genetic Engineering and Biotechnology Center and SOBERANA 02 and SOBERANA Plus, which were developed at Cuba’s Finlay Vaccine Institute (Instituto Finlay de Vacunas Cuba) [@doi:10.37757/mr2022.v24.n3-4.13].
+Here, we focus on the development of the Abdala vaccine, but SOBERANA 01/02/Plus vaccine development program has also met with great success and reported VEs of over 90% in the three-dose regimen [@doi:10.1016/j.lana.2022.100423; @doi:10.1016/j.medj.2022.08.001].
```
Comment (RLordan) [inline content/22.vaccines-trad.md:489]:
```suggestion
Here, we focus on the development of the Abdala vaccine, but SOBERANA 01/02/Plus vaccine development program has also achieved great success and reported VEs of over 90% in the three-dose regimen [@doi:10.1016/j.lana.2022.100423; @doi:10.1016/j.medj.2022.08.001].
```

File: content/22.vaccines-trad.md [hunk 3]
```diff
@@ -424,31 +425,104 @@ These data are retrieved from Our World in Data [@url:https://ourworldindata.org
...
+Notably, Cuba has vaccinated a high percentage of its population, with 87% of the population vaccinated by January 2022 and 90.3% by the end of December 2022 [@url:https://salud.msp.gob.cu/actualizacion-de-la-vacunacion-en-el-marco-de-los-estudios-de-los-candidatos-vacunales-cubanos-y-la-intervencion-sanitaria; @doi:10.37757/MR2022.V24.N1.11].
+Therefore, one consideration in interpreting retrospective cohort studies is that the vaccination rate in Cuba is so high that the two cohorts might not be directly comparable.
+All the same, the fact that the efficacy and effectiveness of the Abdala vaccine have both been estimated to be over 90% against severe illness suggests that this vaccine is highly effective for mitigating the risk of COVID-19.
+As of December 2022, the vaccine had been authorized for distribution in five additional countries, including Mexico and Vietnam, although its evaluation for WHO approval was ongoing [@url:https://extranet.who.int/pqweb/sites/default/files/documents/Status_COVID_VAX_08November2022.pdf; @doi:10.1136/bmj.o230].
+
+However, limited data is available about the Abdala's vaccine's robustness to evolutionary change in SARS-CoV-2.
```
Comment (RLordan) [inline content/22.vaccines-trad.md:515]:
```suggestion
However, limited data is available about the Abdala's vaccine's robustness to evolutionary changes in SARS-CoV-2.
```

**Review by RLordan — APPROVED** [review]
Great additions @rando2, particularly Cuba's vaccine. Thanks a million.

File: content/22.vaccines-trad.md (hunk 1 above)
Comment (rando2) [reply content/22.vaccines-trad.md:467]: They always say VOC/VOI in the study so I was thinking it might be safer to follow suit!

**Pull request merged**